lean/deps/x86_semantics/leanpkg.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "x86_semantics"
+version = "0.1"
+lean_version = "master"
```

Comment:
I don't know what's wrong with my setup, but using an unknown version like "master" causes my lean process to die and give a mysterious error message in emacs. If I locally change this to "3.4.1" then things start working.

Reply:
I'm going to update this to 3.4.1.

Reply:
Could you verify the updated version works for you then merge?